### DISCUSSION TITLE
muncul tanda * dalam [ ] ketika menjalankan shortcut ctrl + enter

### DISCUSSION THREAD

**Discussion by AngelMartha** [teamalgoritma/community#596] [Q&A] [answered]
muncul tanda * dalam [ ] ketika menjalankan shortcut ctrl + enter

**Comment by amaliasf** [comment, marked as answer]
Dear @AngelMartha 

Terkait tanda [*] saat menjalankan cell code, hal tersebut menandakan proses running code masih berlangsung. Boleh silahkan ditunggu terlebih dahulu. Namun apabila prosesnya cenderung sangat lama, bisa di terminate saja dengan Klik Menu **Kernel - Interrupt**.

Semoga membantu dan silahkan balas pesan ini  kalau masih terkendala.
